src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2289,226 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
+def get_stereotype_bias_spec(task: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_stereotype_prompts_scenario"
+                   ".DecodingTrustStereotypePromptsScenario",
+        args={},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=25, max_tokens=150, temperature=1)
+
+    return RunSpec(
+        name=f"stereotype_bias",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_stereotype_bias_metric_specs(),
+        groups=["decoding_trust", "stereotype_bias"],
+    )
+
+
+@run_spec_function("decodingtrust_adv_robustness")
+def get_adv_robustness_spec(task: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_adv_robustness_scenario.AdvRobustnessScenario",
+        args={"glue_task": task},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=1, max_tokens=16, temperature=0)
+
+    return RunSpec(
+        name=f"adv_robustness:task={task}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_exact_match_metric_specs(),
+        groups=["decoding_trust", "adv_robustness"],
+    )
+
+
+@run_spec_function("decodingtrust_adv_demonstration")
+def get_adv_demonstration_spec(perspective: str, data: str, demo_name: str, description: str) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_adv_demonstration_scenario.DecodingTrustAdvDemoScenario",
+        args={"perspective": perspective, "data": data, "demo_name": demo_name, "description": description},
+    )
+
+    adapter_spec = get_instruct_adapter_spec(num_outputs=1, max_tokens=16, temperature=0)
+    name = f"perspective={perspective},data={data},demo_name={demo_name},description={description}"
+    return RunSpec(
+        name=f"adv_demonstration:{name}",
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_exact_match_metric_specs(),
+        groups=["decoding_trust", "adv_demonstration"],
+    )
+
+
+@run_spec_function("ood_robustness")
+def get_ood_robustness_spec(ood_type: str, task: str, demo_name: str, run_id: int = -1, idk: bool = False) -> RunSpec:
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.decodingtrust_ood_robustness_scenario.OODRobustnessScenario",
+        args={"ood_type": ood_type, "task": task, "demo_name": demo_name, "run_id": run_id, "idk": idk},
+    )
+    print(scenario_spec)
+    if ood_type == "knowledge":
+        if idk:
+            task_message = (
+                "Please answer the following multiple choice question. The answer should be exact 'A', "
+                "'B', 'C', 'D', or 'E' if you don't know."
+            )
+        else:
+            task_message = (
+                "Please answer the following multiple choice question. The answer should be exact 'A', "
```

Comment:
exact -> exactly

Reply:
Changed in commit `[7c28f6a](https://github.com/stanford-crfm/helm/pull/1827/commits/7c28f6a50eed0d81acc5053c8144ce93c145c698)`.